### PR TITLE
openstack-test: Enable promotion

### PR DESCRIPTION
**Pull request by pierreprinetti**
Implements [OSASINFRA-2682](https://issues.redhat.com/browse/OSASINFRA-2682)

### CODE REVIEW

**Comment by pierreprinetti** [comment]
Funny that 4.3 does not need the image mapping.

File: ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml [hunk 1]
```diff
@@ -7,13 +7,13 @@ build_root:
 promotion:
   additional_images:
     openstack-tests: bin
-  name: "4.10"
+  name: main
```
Comment (petr-muller) [inline ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml:10]: Keep 4.10 here

File: ci-operator/config/openshift/openstack-test/openshift-openstack-test-release-4.10.yaml [hunk 2]
```diff
@@ -7,7 +7,6 @@ build_root:
 promotion:
   additional_images:
     openstack-tests: bin
-  disabled: true
```
Comment (petr-muller) [inline ci-operator/config/openshift/openstack-test/openshift-openstack-test-release-4.10.yaml:10]: And keep disabled here

**Review by petr-muller — COMMENTED** [review]
main/4.10 are a little special case, related to OCP release cycle and centralized branch management. Right now, OCP is in development phase for 4.10, which means main/master branches promote to 4.10 and release-4.10 branches do not promote anywhere.

https://docs.ci.openshift.org/docs/architecture/branching/#normal-development-for-4x-release

**Comment by pierreprinetti** [comment]
/retest
:thinking:

**Comment by pierreprinetti** [comment]
>  error: unable to create a release: operator "cluster-node-tuning-operator" contained an invalid image-references file: no input image tag named "cluster-node-tuned"

I fail to grasp what makes 4.4 and 4.5 image generation fail
/retest

File: ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml [hunk 3]
```diff
@@ -13,7 +13,7 @@ releases:
   latest:
     integration:
       include_built_images: true
-      name: "4.10"
+      name: main
```
Comment (petr-muller) [inline ci-operator/config/openshift/openstack-test/openshift-openstack-test-main.yaml:16]: This should be `4.10`

**Comment by pierreprinetti** [comment]
Failures in 4.4 and 4.5 image building do not seem related with this change.

**Pull request merged**